core/src/main/scala/org/allenai/common/FileUtils.scala

```diff
@@ -0,0 +1,73 @@
+package org.allenai.common
+
+import _root_.au.com.bytecode.opencsv.CSVReader
+
+import java.io._
+
+import scala.collection.JavaConverters._
+import scala.io.{ BufferedSource, Codec, Source }
+
+/** Various convenient utilities for reading files and resources. */
+object FileUtils extends Logging {
+
+  /** Get a file as a buffered source. */
```

Comment:
This doesn't do anything other than delegate to the `Source` implementation.

Instead of using this method, you should do something like:
```scala
// Use the optional codec if it's provided; else, use the default codec.
val localCodec = myOptionCodec.getOrElse(implicitly[Codec])
val source = Source.fromFile(file)(localCodec)
```
or with implicits:
```scala
// Implicit codec; really should only be used if it's shared in a class.
implicit val localCodec = myOptionCodec.getOrElse(implicitly[Codec])
val source = Source.fromFile(file)
```